webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -39,7 +39,14 @@ class BookmarkContainer extends React.Component {
     ) : (
       <Dropdown pullRight id={controller}>
         <Dropdown.Toggle />
-        <Dropdown.Menu>
+        <Dropdown.Menu className='scrollable-dropdown'>
+          {canCreate && (
+            <MenuItem key="newBookmark" id="newBookmark" onClick={this.onClick.bind(this)}>
```

Comment:
`onClick` should be bind at the constructor with the help of the helpers.

I prefer naming those event as well:
```js
onClick={this.handleNewBookmarkClick}
```